configuration/scripts/options/set_nml.gbox80

```diff
@@ -1,4 +1,5 @@
 ice_ic = 'default'
 grid_type = 'rectangular'
 atm_data_type = 'box2001'
+ocn_data_type = 'calm'
```

Comment:
do we want atm_data_type etc in the gbox* option files?  since we have to specify these in addition to the 'full' options (box2001, boxslotcyl), and they might conflict?

Reply:
It seems that we do not need set_nml.gbox* at all. All of those options would be in the other set_nml (box2001, boxslotcyl, boxsym?), etc.

Reply:
Those option files are used by the scripts, and might even be useful empty.  As discussed, we could keep grid_type in there and move the other flags.

Reply:
I left the ice_ic and grid_type in the gbox* for now. We should check if it is needed for other test suites.

Reply:
Left the gbox* as-is. We will address where to change the forcing updates in later PR.